### PR TITLE
1/documentation

### PR DESCRIPTION
**Pull request by gpavelar**
README.md update and initial .gitignore file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,83 @@
-2020-jun-project1-group2
\ No newline at end of file
+2020-jun-project1-group2
```
Comment (nimmi89) [inline README.md:1]: # Migrating a Wordpress Application to AWS [2020-jun-project1-group2]


Name sounds better

File: README.md [hunk 2]
```diff
@@ -1 +1,83 @@
...
+
+# Project 1
+
+This repository is related to source code of the project 1 of group2.
```
Comment (nimmi89) [inline README.md:5]: We can directly have the description after this. Add the table of contents as well

**Review by nimmi89 — COMMENTED** [review]
Some changes to be done!

**Comment by gpavelar** [comment]
Just submmitted the same file in another branch.

**Pull request closed without merging**